data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/kill_fires_events.cfg

```diff
@@ -0,0 +1,43 @@
+#####
+# API(s) being tested: [kill]fire_event=yes
+##
+# Actions:
+# Create a last breath and die events. Put Alice to the recall list. Kill Bob and Alice, and detect with a variable which events have been triggered for whom.
+##
+# Expected end state:
+# A last breath event and a die event is triggered for Bob but not for Alice.
+#####
+{GENERIC_UNIT_TEST "kill_fires_events" (
+    [event]
+        name=last_breath
+		first_time_only=no
+		[set_variable]
+			name=$unit.id|_breath
+			value=yes
+		[/set_variable]
+    [/event]
+    [event]
+        name=die
+		first_time_only=no
+		[set_variable]
+			name=$unit.id|_die
+			value=yes
+		[/set_variable]
+    [/event]
+    [event]
+        name=start
+		[put_to_recall_list]
+			id=alice
+		[/put_to_recall_list]
+		[kill]
+			id=alice,bob
+			fire_event=yes
+		[/kill]
+        {ASSERT ({VARIABLE_CONDITIONAL bob_die boolean_equals yes})}
+        {ASSERT ({VARIABLE_CONDITIONAL alice_die boolean_not_equals yes})}
+        {ASSERT ({VARIABLE_CONDITIONAL bob_breath boolean_equals yes})}
+        {ASSERT ({VARIABLE_CONDITIONAL alice_breath boolean_not_equals yes})}
+        {SUCCEED}
+    [/event]
+)}
+

```

Comment:
The line endings on this file are DOS style. Running `data/tools/wmlindent FILENAME` on it will clean it up, and also fix this trailing blank line.